modules/apps/portal/portal-cluster-multiple/src/test/java/com/liferay/portal/cluster/multiple/internal/ClusterMasterExecutorImplTest.java

```diff
@@ -694,8 +688,8 @@ public static void waitUntilBlock(int threadCount) {
 		}
 
 		@Around(
-			"execution(protected * com.liferay.portal.cluster.multiple." +
-				"internal.ClusterExecutorImpl.getClusterNodeId(..))"
+			"execution(public * com.liferay.portal.cluster.multiple.internal." +
```

Comment:
move this change for https://github.com/rafaprax/liferay-portal/pull/1349/commits/1663780b60470bdcdeb68016563e04818ddf3319 commit